editoast/Dockerfile

```
@@ -7,6 +7,12 @@ RUN cargo chef prepare --recipe-path recipe.json
 
 FROM chef as builder
 
+# geos crate requirement
+RUN apt update -yqq && \
+    apt install -yqq --no-install-recommends libgeos-dev && \
+    apt-get purge -y --auto-remove -o APT::AutoRemove::RecommendsImportant=false && \
+    rm -rf /var/lib/apt/lists/*
```

Since it's the build stage we don't need to clean up.

```suggestion
RUN apt update -yqq &&  apt install -yqq --no-install-recommends libgeos-dev
```